general_rules/Scenario.tex

```diff
@@ -1,202 +1,122 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{Scenario}
```

Comment:
"Scenario" is a strange word choice here. More accurate would be "Setting" or "Environment", but probably best would just be "Arena".

Reply:
I think scenario is fairly established but I'm willing to change it if other TC members agree.

Reply:
I think "Arena" and "Environment" are better. All of this subsection seems to be about the arena setup.